src/trials.jl

```diff
@@ -316,6 +322,16 @@ function asciihist(bins, height=1)
     map(height -> if height < 1; ' ' else histbars[height] end, heightmatrix)
 end
 
+function hist_round_low(times, lo=minimum(times), av=mean(times))
+    av < 0.1 && return 0.0  # stop at 0, not 0.01 ns, in trivial cases
+    raw = min(lo, av / 1.03)  # demand low edge 3% from mean, or further
+    return round(raw, RoundDown; sigdigits = 2)
```

Comment:
Left edge of the histogram is 0 for trivial cases like `@benchmark 1+1`, minor tidying-up. These will also have a right edge of `1 ns`.

`av / 1.03` guarantees at least 3% below the mean. A similar condition on the right edge means the width is seldom below 10%. Thus benchmarks with very small variation will tend to sit in the middle. I'm not 100% sure that's a good idea, perhaps real use will tell?

Reply:
After using this a bit, I do think seeing a scatter of bars in the centre is a sign that your whole distribution is very narrow, and hence you need not pay too much attention to the histogram.